microservices/first/main.py

```diff
@@ -1,27 +1,28 @@
 # Endpoint documentation: https://github.com/jakejack13/SPEED/blob/microservices/first_api_doc.md
-import json
 from utils import DBManager
 import utils
 
 from flask import Flask, request, jsonify
```

Comment:
```suggestion
from flask import Flask, request, jsonify, Response
```
Shouldn't it be this instead then?

Reply:
Then the type annotation is just `Response`